contrib/ec2/deis.template.json

```diff
@@ -298,6 +298,12 @@
             "Protocol": "HTTP"
           },
           {
+            "InstancePort": "443",
+            "InstanceProtocol": "HTTPS",
+            "LoadBalancerPort": "443",
+            "Protocol": "HTTPS"
+          },
```

Comment:
Yeah, actually that's [exaclty what the official Deis docs suggest](http://docs.deis.io/en/latest/managing_deis/ssl-endpoints/#installing-the-ssl-certificate)